### PR TITLE
Problem: Ported tx-query not tested

### LINKED ISSUE #1235: Problem: ported tx-query not tested
integration tests need to launch IAS proxy + new tx-query

1. multiple tx-query in multi-node -- needs port configuration workaround for EDP
2. extra flag for linker for building in Docker, but that needs to be reset back or tx-query excluded from unit tests
3. some decoding issues -- potentially different crate version

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Added `tx-query-next` in integration tests. Fixes #1235.

### CODE REVIEW

**Comment by devashishdxt** [comment]
bors try

**Comment by devashishdxt** [comment]
bors try

**Comment by devashishdxt** [comment]
bors cancel

**Comment by devashishdxt** [comment]
bors try

**Comment by devashishdxt** [comment]
bors try-

**Comment by devashishdxt** [comment]
bors try

File: build_tx_query.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+cargo build
+cd -
+
+# Build enclave app
+cd $APP_PATH
+cargo build
+cd -
+
+# Convert the app
+ftxsgx-elf2sgxs target/x86_64-fortanix-unknown-sgx/debug/$APP_NAME --heap-size 0x2000000 --stack-size 0x20000 --threads 6 --debug
```
Comment (tomtau) [inline build_tx_query.sh:20]: are those heap and stack sizes enough?
Stack is different for https://github.com/crypto-com/chain/blob/master/chain-tx-enclave/tx-query/enclave/Enclave.config.xml

Also one may need to sign
https://edp.fortanix.com/docs/tasks/deployment/#Signing
with the same debug/testing key: https://github.com/crypto-com/chain/blob/master/chain-tx-enclave/tx-query/enclave/Enclave_private.pem

**Comment by devashishdxt** [comment]
Closing this for now.

**Pull request closed without merging**